cardano-node-chairman/test/Test/Cardano/Node/Chairman.hs

```diff
@@ -72,17 +88,18 @@ prop_spawnOneNode = H.propertyOnce . H.workspace "temp/chairman" $ \tempDir -> d
     hNodeStdout <- H.evalM . liftIO $ IO.openFile nodeStdoutFile IO.WriteMode
     hNodeStderr <- H.evalM . liftIO $ IO.openFile nodeStderrFile IO.WriteMode
 
+    H.diff (L.length socketFile) (<=) H.maxUnixDomainSocketNameLength
```

Comment:
Sanity assertion that the socket file is a valid Unix Domain Socket name, otherwise there is no point in continuing.